src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
Slightly lost. Possibly due to typo.
Lemme try to sum up.

It feels weird for the `RecordCall` (`RecordedCalls` in your comment?) _method_ to make a change that's visible in the `AllRecordedCalls` _property_.